pptx/package.py

```diff
@@ -74,7 +74,7 @@ def next_image_partname(self, ext):
         def first_available_image_idx():
             image_idxs = sorted([
                 part.partname.idx for part in self.iter_parts()
-                if part.partname.startswith('/ppt/media/image')
+                if part.partname.startswith('/ppt/media/image') and part.partname.idx is not None
```

Comment:
Using the files provided, you can reproduce the error, when using Python 3.

My guess is that it breaks if you have more than one image in master template. So they are probably listed by self.iter_parts() and have image type, but have None for partname.idx ?

```
import pptx
from pptx.util import Inches, Pt

presentation = pptx.Presentation('template.pptx')
blank_slide_layout = presentation.slide_layouts[0]
slide = presentation.slides.add_slide(blank_slide_layout)

shapes = slide.shapes
shapes.placeholders[0].text = 'dummy text'
shapes.add_picture('test_image.png', left=Inches(0.1), top=Inches(1.1), height=Inches(3.4))

```
[template.pptx](https://github.com/scanny/python-pptx/files/293189/template.pptx)
<img width="900" alt="test_image" src="https://cloud.githubusercontent.com/assets/5244286/15706762/da08ff2e-27fd-11e6-8fcc-2ab9a3f8ce72.png">